hudi-utilities/src/main/java/org/apache/hudi/utilities/HoodieCompactor.java

```diff
@@ -295,4 +301,11 @@ private String getSchemaFromLatestInstant() throws Exception {
     Schema schema = schemaUtil.getTableAvroSchema(false);
     return schema.toString();
   }
+
+  private void cleanAfterCompact(SparkRDDWriteClient client) {
+    if (client.getConfig().isAutoClean()) {
+      LOG.info("Start to clean synchronously.");
```

Comment:
Can the user set up auto clean manually? If not, it is always true.

Reply:
Auto-Clean can be modified through the input para config of the job constructor.